cdk/jest.setup.js

```diff
@@ -1 +1,2 @@
-jest.mock("@guardian/cdk/lib/constants/tracking-tag");
+jest.mock('@guardian/cdk/lib/constants/tracking-tag');
+jest.mock('@guardian/private-infrastructure-config');
```

Comment:
Adds the Jest mock - https://github.com/guardian/private-infrastructure-config#using-with-jest.

Reply:
Thanks, I add missed this!